targets/unity-v2/source/Shared/Internal/OneDsUtility.cs

```diff
@@ -21,49 +22,74 @@ public static void ParseResponse(long httpCode, Func<string> getText, string err
             else
             {
                 string httpResponseString;
+
                 try
                 {
                     httpResponseString = getText.Invoke();
                 }
                 catch (Exception exception)
                 {
-                    var error = new PlayFabError();
-                    error.Error = PlayFabErrorCode.ConnectionError;
-                    error.ErrorMessage = exception.Message;
-                    callback?.Invoke(error);
+                    callback.Invoke(new OneDsError
+                    {
+                        Error = PlayFabErrorCode.ConnectionError,
+                        ErrorMessage = exception.Message
+                    });
                     return;
                 }
 
                 if (httpCode >= 200 && httpCode < 300)
                 {
+                    Action<Exception> printErrorMessage = e =>
```

Comment:
would be nice to see a more detailed named parameter instead of just "e".

Reply:
Why would we want to change the errorMsg here? UnityEngine.Debug.LogException already accepts an Exception which we could plug right in. I'm assuming that may look gross, hence explicilty calling out the source of the error and the message.